rules/duo_rules/duo_user_anomalous_push.py

```diff
@@ -0,0 +1,22 @@
+from panther_base_helpers import deep_get
+
+
+def rule(event):
+    return event.get("reason") == "anomalous_push" and event.get("result") == "denied"
+
+
+def title(event):
+    # pylint: disable=line-too-long
+    return f"Duo Auth denied due to an anomalous 2FA push for {deep_get(event, 'user', 'name', default='<Unknown>')}"
+
+
+def alert_context(event):
+    return {
+        "factor": event.get("factor"),
+        "reason": event.get("reason"),
+        "user": deep_get(event, "user", "name"),
```

Comment:
`deep_get` should have a `default=` kwarg providing a default. alert_context will get cast to a string via `json.dumps()` in some scenarios.

Reply:
Done 👍

Reply:
Done 👍